src/css/brandColors.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable n/no-sync */
+/* eslint-disable jsdoc/require-jsdoc */
+import path from 'path';
+
+import { getCssColorVariables } from '../../utils';
+import jsonData from '../figma/brandColors.json';
```

Comment:
Can we merge the first version of this PR to test the current CSS variables agains the current token json
```suggestion
import jsonData from '../figma/tokens.json';
```